tutorials/migration_sdf.md

```diff
@@ -269,8 +269,8 @@ that worked on Gazebo classic may need more plugins on Gazebo.
 ## Materials
 
 Gazebo does not support Ogre material files like Classic does, because Gazebo
-Gazebo can be used with multiple rendering engines. Therefore, materials defined
-within a `<script>` aren't supported on Gazebo, for example:
+can be used with multiple rendering engines. For example, materials defined
+within a `<script>` aren't supported on Gazebo:
```

Comment:
Saying they aren't supported here is somewhat confusing even though further below, we mention that there's some level of support.

```suggestion
can be used with multiple rendering engines. There is limited support for pre-defined materials, but arbitrary materials defined within a `<script>` aren't supported:
```